Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle.yml

```
@@ -20,11 +20,69 @@
+  - type: Appearance
+  - type: SpentAmmoVisuals
+  - type: StaticPrice
+    price: 10
+
 - type: entity
   id: CartridgeMinigun
   name: cartridge (.10 rifle)
-  parent: BaseCartridgeHeavyRifle
+  parent: LPPBaseCartridgeMinigun
   components:
   - type: CartridgeAmmo
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)